mitodl/bootcamp-ecommerce
# Added integration for bootcamp admission

#### What are the relevant tickets?
closes #57 

#### What's this PR do?
Adds integration for the Bootcamp Admissions web app

#### How should this be manually tested?

1. set the proper environment variables for the test website
2. create a user with the same email on your edx and the bootcamp admissions portal
3. set up your user as admitted in the bootcamp admissions portal
4. verify that you see the klass to pay in the bootcamp ecommerce
